CONTRIBUTING.md

```diff
@@ -1016,15 +1016,46 @@ To fix it:
    superset db heads
    ```
 
-   This should list two or more migration hashes.
+   This should list two or more migration hashes. E.g.
+
+   ```bash
+   1412ec1e5a7b (head)
+   67da9ef1ef9c (head)
+   ```
+
+2. Pick one of them as the parent revision, open the script for the other revision
+   and update `Revises` and `down_revision` to the new parent revision. E.g.:
+
+   ```diff
+   --- a/67da9ef1ef9c_add_hide_left_bar_to_tabstate.py
+   +++ b/67da9ef1ef9c_add_hide_left_bar_to_tabstate.py
+   @@ -17,14 +17,14 @@
+   """add hide_left_bar to tabstate
+
+   Revision ID: 67da9ef1ef9c
+   -Revises: c501b7c653a3
+   +Revises: 1412ec1e5a7b
+   Create Date: 2021-02-22 11:22:10.156942
+
+   """
+
+   # revision identifiers, used by Alembic.
+   revision = "67da9ef1ef9c"
+   -down_revision = "c501b7c653a3"
+   +down_revision = "1412ec1e5a7b"
+
+   import sqlalchemy as sa
+   from alembic import op
+   ```
 
-1. Create a new merge migration
+   Alternatively you may also run `superset db merge` to create a migration script
+   just for merging the heads.
 
    ```bash
    superset db merge {HASH1} {HASH2}
    ```
 
-1. Upgrade the DB to the new checkpoint
+3. Upgrade the DB to the new checkpoint
```

Comment:
nit: you should be able to always use number `1.` and it will automatically increment it on the next bullet (avoids the need for updating all bullets when inserting/deleting bullets)

Reply:
Yeah, I am aware of this markdown feature, just thought it reads nicer in the source code. The numbers doesn't have to be always `1` either, so one can also just choose not to update them while inserting/deleting bullets. I happen to think it's not a big deal to update it here.

E.g.  this also works:

```
1.
2.
2.
5.
```